extensions/commands/art/cmd_build_info.py

```diff
@@ -389,7 +389,8 @@ def build_info_create(conan_api: ConanAPI, parser, subparser, *args):
     data = load_json(args.json)
 
     # remove the 'conanfile' node
-    data["graph"]["nodes"].pop("0")
+    if data["graph"]["nodes"]["0"]["ref"] == "conanfile":
```

Comment:
I am concern about cases where the consumer might have defined a ``name`` and ``version``, so the ``ref`` might not be "conanfile", but the actual ``name/version``. 

Maybe it would be necessary to add a test for that scenario.

Reply:
I tried to replay the scenario on the commandline using the following commands:
```bash
conan new cmake_lib -d name=mypkg -d version=1.0 --force
conan create . --format json -tf='' -s build_type=Release > create_release.json
```

That results in the following `create_release.json` (I only pasted the 1st lines):
```json
{
    "graph": {
        "nodes": {
            "0": {
                "ref": "conanfile",
                "id": "0",
```

Is this the behavior you were thinking of for a test?